production/promtail-mixin/dashboards.libsonnet

```diff
@@ -4,23 +4,22 @@ local loki_mixin_utils = import 'loki-mixin/dashboards/dashboard-utils.libsonnet
 local utils = import 'mixin-utils/utils.libsonnet';
 
 {
-  grafanaDashboards+: {
+  grafanaDashboards+:: {
     local dashboard = (
       loki_mixin_utils {
         _config+:: configfile._config,
       }
     ),
     local dashboards = self,
 
-    // local labelsSelector = 'cluster=~"$cluster", namespace=~"$namespace"',
     local labelsSelector = dashboard._config.per_cluster_label + '=~"$cluster", namespace=~"$namespace"',
     local quantileLabelSelector = dashboard._config.per_cluster_label + '=~"$cluster", job=~"$namespace/promtail"',
 
     'promtail.json': {
                        local cfg = self,
                      } +
                      dashboard.dashboard('Loki / Promtail', uid='promtail')
-                     .addCluster()
+                     .addTemplate('cluster', 'promtail_build_info', dashboard._config.per_cluster_label)
```

Comment:
`addCluster()` was looking for `loki_build_info`, so if a cluster didn't run Loki, it wouldn't appear.